tests/python/pants_test/bin/test_loader_integration.py

```diff
@@ -36,3 +36,11 @@ def test_alternate_entrypoint_not_callable(self):
     self.assert_failure(pants_run)
     self.assertIn('TEST_STR', pants_run.stderr_data)
     self.assertIn('not callable', pants_run.stderr_data)
+
+  def test_alternate_entrypoint_scrubbing(self):
+    pants_run = self.run_pants(
+      command=['help'],
+      extra_env={'PANTS_ENTRYPOINT': 'pants.bin.pants_exe:test_env'}
+    )
+    self.assert_success(pants_run)
+    self.assertIn('None', pants_run.stdout_data)
```

Comment:
This assertion seems likely to accidentally match the wrong thing. Would be good to match something more explicit, like `PANTS_ENTRYPOINT=None`, or something.

Reply:
done